images/router/haproxy/conf/haproxy-config.template

```diff
@@ -186,7 +202,23 @@ frontend public
 # determined by the next backend in the chain which may be an app backend (passthrough termination) or a backend
 # that terminates encryption in this router (edge)
 frontend public_ssl
-  bind :::{{env "ROUTER_SERVICE_HTTPS_PORT" "443"}} v4v6 {{ if matchPattern "true|TRUE" (env "ROUTER_USE_PROXY_PROTOCOL" "") }} accept-proxy{{ end }}
+  {{ if ne (env "ROUTER_SERVICE_IP_ADDRESSES" "") "" }}
+    {{ resetBindline }}
+    {{ with $values := split (env "ROUTER_SERVICE_IP_ADDRESSES" "")  "," }}
+      {{ $bindLine := "" }}
+      {{ range $idx, $ipaddr := $values }}
+        {{- if and (gt $idx 0) ( gt ( len $ipaddr ) 0 )  }}
+          {{ $bindLine := createBindline  $ipaddr (env "ROUTER_SERVICE_HTTPS_PORT" "443") }}
```

Comment:
You mean something like this in the range loop.

```
bind {{ $ipaddr }}:{{ env "ROUTER_SERVICE_HTTPS_PORT" "443" }} v4v6 {{ if matchPattern "true|TRUE" (env "ROUTER_USE_PROXY_PROTOCOL" "") }} accept-proxy{{ end }}
```

I will test it and update the pr